server/service/src/programs/patient/patient_updated.rs

```diff
@@ -82,9 +83,11 @@ pub fn update_patient_row(
         r#type: NameType::Patient,
         is_customer: existing_name.map(|n| n.is_customer).unwrap_or(true),
         is_supplier: existing_name.map(|n| n.is_supplier).unwrap_or(false),
-        supplying_store_id: existing_name.and_then(|n| n.supplying_store_id.clone()),
-        first_name: first_name,
-        last_name: last_name,
+        supplying_store_id: existing_name
```

Comment:
Can you add a comment please: // supplying_store_id is the home store for patient, needed for compatibility with mSupply